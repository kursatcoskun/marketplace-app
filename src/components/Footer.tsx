import styled from "styled-components";

export interface FooterProps {}

const Footer = styled.div`
  position: fixed;
  left: 0;
  bottom: 0;
  width: 100%;
  height: 40px;
  background-color: #e5e5e5;
  color: #1ea4ce;
  text-align: center;
`;

export default Footer;
