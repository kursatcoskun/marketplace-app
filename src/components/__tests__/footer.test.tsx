// MyComponent.test.js
import React from "react";
import { shallow } from "enzyme";
import Footer from "../Footer";
describe("FooterComponent", () => {
  it("should render footer component", () => {
    const wrapper = shallow(<Footer />);
  });

  it("should render initial layout", () => {
    const component = shallow(<Footer />);
    expect(component.getElements()).toMatchSnapshot();
  });
});
