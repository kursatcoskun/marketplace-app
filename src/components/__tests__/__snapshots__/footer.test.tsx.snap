// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FooterComponent should render initial layout 1`] = `
Array [
  <div
    className="sc-bdnxRM lhfUZN"
  />,
]
`;
